database/mssql/setup.sh

```diff
@@ -1,22 +1,30 @@
 #!/bin/bash
 
 # wait for MSSQL server to start
-export STATUS=1
-i=0
+pid=$!
 
-while [[ $STATUS -ne 0 ]] && [[ $i -lt 30 ]]; do
-	i=$i+1
-	/opt/mssql-tools/bin/sqlcmd -t 1 -U sa -P $MSSQL_SA_PASSWORD -Q "select 1" >> /dev/null
-	STATUS=$?
+echo "Waiting for MS SQL to be available ⏳"
+/opt/mssql-tools/bin/sqlcmd -l 30 -S localhost -h-1 -V1 -U sa -P $MSSQL_SA_PASSWORD -Q "SET NOCOUNT ON SELECT \"YAY WE ARE UP\" , @@servername"
+is_up=$?
+while [ $is_up -ne 0 ] ; do
+	echo -e $(date)
+	/opt/mssql-tools/bin/sqlcmd -l 30 -S localhost -h-1 -V1 -U sa -P $MSSQL_SA_PASSWORD -Q "SET NOCOUNT ON SELECT \"YAY WE ARE UP\" , @@servername"
+	is_up=$?
+	sleep 5
 done
 
-if [ $STATUS -ne 0 ]; then
-	echo "Error: MSSQL SERVER took more than thirty seconds to start up."
-	exit 1
-fi
+echo "======= MSSQL SERVER STARTED ========"
+# Run every script in /scripts
```

Comment:
I've kept this section in as it will allow us to add additional scripts overtime easily.